contents/blog/product-led-growth.md

```diff
@@ -0,0 +1,193 @@
+---
+date: 2020-08-18
+title: Why You May Not Need a Sales Team
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+<!-- ![PLG Banner](../images/plg-header.png) -->
+
+Let's say you are the founder of a new tech startup. 
+
+A few months back, you and a couple of friends (all developers) built an amazing tool and have now received some Seed funding. 
```

Comment:
is the repeated use of 'some' in this line and the next deliberate as a style thing?